core/shared-preferences/src/main/java/com/appcoins/wallet/sharedpreferences/OemIdPreferencesDataSource.kt

```diff
@@ -18,8 +18,13 @@ class OemIdPreferencesDataSource @Inject constructor(
       .putString(CURRENT_OEMID, value)
       .apply()
 
-  fun getCurrentOemId() = sharedPreferences.getString(CURRENT_OEMID, "") ?: ""
```

Comment:
This is to get the content of wallet to not be seen by user/miner whilst it is moved out, correct? Or am I wrong?